### ISSUE TITLE
name clash with other Fakturama project

### ISSUE THREAD

**Issue by rheydenr** [Fakturama/Fakturama#1]
Hi repository owner,
I'm the project owner of another project which is also called "Fakturama". You can visit it here: www.fakturama.info (in German). Since there's obviously no activity in this repository I'd ask you to take over this account so it fits better to my project. Please contact me for further information. Thanks in advance,

Ralf.

**Comment by meladelio** [comment]
There is no clash at all, Mr. Ralf. You just went back f
where you started. Welcome back to the roots, or should it sound like, our roots?

**Comment by rheydenr** [comment]
Peter???

**Comment by meladelio** [comment]
Klar, der Weihnachtsmann... hat Urlaub.                                                                                                                                                                                                                                                                                                                                        Gesendet von meinem BlackBerry 10-Smartphone.                                                                                                                                                                                                                Von: Ralf HeydenreichGesendet: Montag, 20. November 2017 23:30An: Fakturama/FakturamaAntwort an: Fakturama/FakturamaCc: meladelio; CommentBetreff: Re: [Fakturama/Fakturama] name clash with other Fakturama project (#1)Peter???

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/Fakturama/Fakturama","title":"Fakturama/Fakturama","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/Fakturama/Fakturama"}},"updates":{"snippets":[{"icon":"PERSON","message":"@rheydenr in #1: Peter???"}],"action":{"name":"View Issue","url":"https://github.com/Fakturama/Fakturama/issues/1#issuecomment-345853456"}}}

**Comment by rheydenr** [comment]
Dann genieß mal Deinen Urlaub weiter. Da richtest Du vielleicht nicht ganz so viel Schaden an...

**Comment by meladelio** [comment]
Gewiss, deine Weitsichtigkeit spricht Baende, hu hu hu ;-)                                                                                                                                                                                                                                                                                                                                        Gesendet von meinem BlackBerry 10-Smartphone.                                                                                                                                                                                                                Von: Ralf HeydenreichGesendet: Dienstag, 21. November 2017 00:57An: Fakturama/FakturamaAntwort an: Fakturama/FakturamaCc: meladelio; CommentBetreff: Re: [Fakturama/Fakturama] name clash with other Fakturama project (#1)Dann genieß mal Deinen Urlaub weiter. Da richtest Du vielleicht nicht ganz so viel Schaden an...

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/Fakturama/Fakturama","title":"Fakturama/Fakturama","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/Fakturama/Fakturama"}},"updates":{"snippets":[{"icon":"PERSON","message":"@rheydenr in #1: Dann genieß mal Deinen Urlaub weiter. Da richtest Du vielleicht nicht ganz so viel Schaden an..."}],"action":{"name":"View Issue","url":"https://github.com/Fakturama/Fakturama/issues/1#issuecomment-345871190"}}}
